src/content/docs/journey-app-slow/my-app-is-slow.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I like the layout of this doc but it's confusing to me to have it as a standalone doc in the nav rather than in the category. I also think if it continues to exist it needs at least some kind of intro ¶ to contextualize it for the reader.

Reply:
Whoops this is just a leftover from experimenting when we had that one call. Will delete!